developer-docs-site/docs/concepts/state-sync.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "State Synchronization"
+slug: "state-sync"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# State Synchronization
+
+Nodes in an Aptos network, both the validator nodes and the fullnodes, must always be synchronized to the latest Aptos blockchain state. The state synchronization (state sync) component that runs on each node is responsible for this synchronization. To achieve this synchronization, the state sync identifies and fetches new blockchain data from the peers, validates the data and persists it to the local storage.
+
+## State sync modes
+
+If you are a node operator, you can configure the state sync mode to operate in one of the two synchronization modes, depending on the state of the node, i.e., whether the node has just started (bootstrapping mode) or has been running for some time (continuous sync mode). 
+
+### Bootstrapping mode
+
+When the node starts, the state sync should be in the bootstrapping mode so that the node can catch up to the Aptos blockchain. There are three bootstrapping modes:
```

Comment:
"When the node starts, the state sync should be in the bootstrapping mode so that the node can catch up to the Aptos blockchain. There are three bootstrapping modes:" -> "When the node starts, state sync will perform bootstrapping by using the specified bootstrapping mode configuration. This allows the node to catch up to the Aptos blockchain. There are three bootstrapping modes:"